views/accessibility-statement.html

```diff
@@ -121,7 +121,7 @@ <h2 class="govuk-heading-m" id="were-doing">
             <p class="govuk-body"><a class="govuk-link" href="https://companieshouse.blog.gov.uk/category/user-research/">Read our blogs about usability</a>
                 to find out more about our user research work.</p>
 
-            <p class="govuk-body">This statement was prepared on 22 July 2022. It was last updated on 15 April 2024.</p>
+            <p class="govuk-body">This statement was prepared on 22 July 2022. It was last updated on 28 October 2024.</p>
```

Comment:
Guessed at 28th October here but will check what @sarahocarroll and @pvoddenCH think.

Reply:
Last time we made a change to the statement (for remove) we allowed a few extra days for testing so that the date would be as close to the live date as possible.  As @sarahocarroll and I are on leave until 4th November and its unlikely to go live before then, I think it might be worth adding a few extra days

Reply:
👍 will do. I'll change to 4th November.

Worth noting that the main CHS one for this change isn't LIVE yet (ticket on our 191 release notes) and it has 11th October...

See https://github.com/companieshouse/ch.gov.uk/pull/479.

Reply:
Release 191 is live... But the tickets weren't updated... I'll do that now. 
 https://find-and-update.company-information.service.gov.uk/help/accessibility-statement is showing the expected updated v2.2 and the 11th October... 

Putting our date as the 4th November will be fine.

Reply:
Date changed.